index.ts

```diff
@@ -170,15 +191,71 @@ async function main() {
   });
   console.log(safeOpHash);
 
-  const signature = await getNearSignature(safeOpHash);
+  // const signature =
+  //   "0xef028109f94aabaf9cd4fb7f2cce311527567341d3439a40fe15e6e01c45249b790e5e5c1f0803f668219f97a95947a982e4f88d8f9abc675adcdb926ed7504d1c";
+  const signature = await getNearSignature(nearAdapter, safeOpHash);
   console.log(
     await sendUserOperation(
-      { ...unsignedUserOp, signature },
+      asERC4337Payload({ ...unsignedUserOp, signature }),
       await contracts.entryPoint.getAddress(),
     ),
   );
 }
 
+interface UserOpData {
+  sender: string;
+  nonce: string;
+  factory?: string | ethers.Addressable;
+  factoryData?: string | ethers.Addressable;
```

Comment:
Types are wrong here, factoryData are bytes and not an address.

Reply:
I got this from the type hints. Might have something to do with what ethers is returning on their construction. Will double check.

Reply:
Although it might be technically true that they should be bytes. By construction they are as stated here:

<img width="817" alt="Screenshot 2024-06-13 at 13 20 49" src="https://github.com/bh2smith/nearly-safe/assets/11778116/dc60d6f8-a26a-409d-84fd-edbcd101b76f">

Reply:
`factory` is an `Addressable`, `factoryData` is not.